### ISSUE TITLE
Error when starting app

### ISSUE THREAD

**Issue by dpdanpittman** [nicholas-2/steem-state#3]
I just started seeing this error when starting my app.

```2020-04-07T10:29:20.200339+00:00 app[web.1]: (node:23) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'transactions' of null
2020-04-07T10:29:20.200367+00:00 app[web.1]:     at processBlock (/app/processor.js:93:30)
2020-04-07T10:29:20.200368+00:00 app[web.1]:     at /app/processor.js:44:11
2020-04-07T10:29:20.200369+00:00 app[web.1]:     at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-04-07T10:29:20.200490+00:00 app[web.1]: (node:23) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
2020-04-07T10:29:20.200612+00:00 app[web.1]: (node:23) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

**Comment by charette-sh** [comment]
Sorry I took so long to respond.

I can’t tell right now what that error is because it’s wrapped in a promise. So I have released v1.3.7 of `steem-state` which *should* make it print out the error. The new commit was aa4ca19.

(Note: you don’t have to actually add the callback function; it will print out the error with or without the function)

**Comment by dpdanpittman** [comment]
Ok no problem.  Seems to be working again now.  Not sure what was up.  Can
you also pull the request from disregardfiat please. Im currently using the
changes in my code.
